### PR TITLE
Update controlling_concurrency.md

### PR DESCRIPTION
**Pull request by stephanieatte**
Clarification about Concurrency groups and job state

### CODE REVIEW

File: pages/pipelines/controlling_concurrency.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You can add concurrency limits to steps either through Buildkite, or your `pipel
...
 
-A concurrency group works like a queue; it returns jobs in the order they entered the queue (oldest to newest). The concurrency group only cares about jobs in "active" states. and the group becomes "locked" when the concurrency limit for jobs in these states is reached. Once a job moves from an active state to a terminal state (`finished` or `cancelled`) the job is removed from the queue, opening up a spot for another job to enter.
+A concurrency group works like a queue; it returns jobs in the order they entered the queue (oldest to newest). The concurrency group only cares about jobs in "active" states. and the group becomes "locked" when the concurrency limit for jobs in these states is reached. Once a job moves from an active state to a terminal state (`finished` or `cancelled`) the job is removed from the queue, opening up a spot for another job to enter. If a Job’s State is `LIMITED` then it is waiting for another job, ahead of it  in the same Concurrency Group to finish.
```
Comment (mbelton-buildkite) [inline pages/pipelines/controlling_concurrency.md:23]: Style fixes:

```suggestion
A concurrency group works like a queue; it returns jobs in the order they entered the queue (oldest to newest). The concurrency group only cares about jobs in "active" states, and the group becomes "locked" when the concurrency limit for jobs in these states is reached. Once a job moves from an active state to a terminal state (`finished` or `canceled`), the job is removed from the queue, opening up a spot for another job to enter. If a job's state is `limited`, it is waiting for another job ahead of it in the same concurrency group to finish.
```

**Comment by stephanieatte** [comment]
@mbelton-buildkite I made the suggested changes

**Review by mbelton-buildkite — APPROVED** [review]
Looks good to me :)

**Comment by mbelton-buildkite** [comment]
I'll merge in the latest `main` to fix the linting for you

**Pull request merged**